frontend/packages/kubevirt-plugin/src/components/vms/vm-console.tsx

```diff
@@ -91,18 +94,30 @@ const VMConsoles: React.FC<VMConsolesProps> = ({
   ) : null;
 
   return (
-    <div className="co-m-pane__body">
-      <AccessConsoles preselectedType={VNC_CONSOLE_TYPE} disconnectByChange={false}>
+    <>
+      {showOpenInNewWindow && (
+        <div className="vm-console-actions">
+          <Button
+            component="a"
+            target="_blank"
+            variant="secondary"
+            href={`/k8s/ns/${namespace}/virtualmachineinstances/${vmName}/standaloneconsole?type=${type.toString()}`}
```

Comment:
we should use the least number of params when we can, to make life easier for our users (eg when they want to share the link).

Could we discard `type` param when VNC is selected as it will fallback to that?

We could mark the defaultness in ConsoleType so it can be reused also in VMConsoleFirehose.

Reply:
we don't know the state of the console type
its inside a PF component
the new button will always open a new window with the type in the URL or fallback to vnc.
We need to add the type here

Reply:
we have a sure fallback to VNC, so we can remove the type in some cases at least. I see, maybe we should not pass the type at all if we cannot guarantee what is selected?

Reply:
I disagree.
Get the type from the URL or fallback to VNC.
Having the type in the URL also helps exposing the ability to change the type to the user, so he gets to know it and change it if he wants.

Reply:
I don't mind that much either way, but we might be exposing links which might not work in the future (when the VM definition changes)